README.md

```diff
@@ -7,19 +7,42 @@ Import your server logs in Piwik with this powerful and easy to use tool.
 * Python 2.6 or 2.7. Python 3.x is not supported.
 * Piwik >= 2.14.0
 
-## Contributors
+Build status (master branch) [![Build Status](https://travis-ci.org/piwik/piwik-log-analytics.svg?branch=master)](https://travis-ci.org/piwik/piwik)
+
+## Supported log formats
+
+The script will import almost all standard web server log files, and many others. The following log formats are supported:
```

Comment:
"almost all standard web server log files, and many others" => "all standard web server log files, and some files with non-standard formats"